src/backend/src/services/work-packages.services.ts

```diff
@@ -729,4 +729,15 @@ export default class WorkPackagesService {
 
     return workPackageTemplateTransformer(workPackage);
   }
+
+  static async getAllWorkPackageTemplates(submitter: User): Promise<WorkPackageTemplate[]> {
+    if (isGuest(submitter.role)) {
+      throw new AccessDeniedGuestException('You must be at least a member to access this function.');
```

Comment:
can you fix the capitalization of your error message, so it looks better: 
![image](https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/7ae85a6d-dd22-473d-b7d3-a5e544a776a0)